code/firmware/rosco_m68k_firmware/duart.asm

```diff
@@ -161,8 +161,9 @@ INITDUART_ATBASE:
     ; for this on the first access, and if we get one, just bail immediately.
     move.b  #0,BERR_FLAG              ; Zero bus error flag
 
-    move.l  $8,BERR_SAVED             ; Save the original bus error handler
-    move.l  #BERR_HANDLER,$8          ; Install temporary bus error handler
+    move.l  $8,BERR_SAVED                   ; Save the original bus error handler
+    move.l  #.POST_WRITE,BERR_CONT_ADDR     ; In case we're on 68000, give a return address...
+    move.l  #BERR_HANDLER,$8                ; Install temporary bus error handler
```

Comment:
This isn't a problem with this PR, so I'll approve regardless, but would it make sense to use `INSTALL_TEMP_BERR_HANDLER` here and in VDP probing?
Right now the code is split between doing the work (clearing `BERR_FLAG`, setting up `BERR_HANDLER`) inline and calling `INSTALL_TEMP_BERR_HANDLER`, so I think that it'd make sense to standardize on one of the two approaches.

Reply:
That's a good shout, might as well do a bit of boyscouting while I'm in this area 👍 . Done.